### DISCUSSION TITLE
Change site name/ change SSL

### DISCUSSION THREAD

**Discussion by Th0rHere** [EasyEngine/easyengine#1624] [Support Requests]
Just checking if their is an easy way to change a site domain. I setup a site using EE (WP) and I do plain on using this domain in the future, but mainly want to get old domain moved onto new server in the meantime so want to stick with original domain for now.

Just checking if anyone has had any luck changing the site name and if their is a method for EE to do it, or does it have to be done manually.

**Comment by Th0rHere** [comment]
Should mention, the reason I want to change the site name and only reason, is to enable SSL, but it was initially setup up using a different name, so I can't use EE straight forward setup as it creates it for that domain, and not the original I want to use.

So to put in context.

OLDDOMAIN.COM on original server.

NEWDOMAIN.COM setup on new sever using EE.

I plan to use NEWDOMAIN.COM for this website and the content, but for now I need to allow it to continue using OLDDOMAIN.COM. However currently the only method for setting up SSL is by using EE command and that creates it for NEWDOMAIN.COM

**Comment by Th0rHere** [comment]
Okay, found out we can add alias, but it doesn't automatically add ssl for the alias, at least it's not renewing SSL, possibly because SSL is not wildcard enabled, but also doesnt seem to be a way to remove ssl and add it again.

**Comment by marcoandrei** [comment]
@Th0rHere 

There is no easy way to do that. You need to create a new site and migrate your current site to it. Look at https://github.com/EasyEngine/easyengine/discussions/1770.

Saúde!
